src/edtr-io/plugins/equations/editor.tsx

```diff
@@ -198,13 +199,12 @@ export function EquationsEditor(props: EquationsProps) {
 
                       return (
                         <ExplanationTr
-                        /* TODO: investigate why this is not working */
-                        /*onFocus={() =>
```

Comment:
@Entkenntnis could you check if this works for you now?
I'm not 💯 % sure I understood the problem :)

Reply:
thank you, it's working fine